.github/workflows/tests_emulator.yml

```
@@ -24,15 +24,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # Refactor to make these dynamic with a low/high bracket only on schedule, not push
-        # For now this is the latest supported API. Previously API 29 was fastest.
-        # #13695: This was reverted to API 30, 31 was unstable. This should be fixed
-        api-level: [30]
-        arch: [x86_64]
-        target: [google_apis]
-        first-boot-delay: [600]
-        # This is useful for benchmarking, do 0, 1, 2, etc (up to 256 max job-per-matrix limit) for averages
-        iteration: [0]
+        # Control which API level + target combinations run
+        include:
+          - api-level: 30
+            arch: x86_64
+            target: google_apis
+            first-boot-delay: 600
+            iteration: 0
+          - api-level: 35
+            arch: x86_64
+            target: google_apis
```

Fantastic - approved, pending upstream PR merge+release and this change here

```suggestion
            target: google_apis_ps16k
```

- Pending https://github.com/ReactiveCircus/android-emulator-runner/pull/440

---

Yep I have amended the commit already, waiting for the release to p force push!

---

had a review comment on that PR I just had time to address now, hopefully will release soon - maintainer there is collaborating fine and that change is needed of course

---

Hasn't been released yet, but temporarily depending on this commit hash for the android-emulator-runner package should work, the PR is merged now https://github.com/ReactiveCircus/android-emulator-runner/commit/66283c03190cd3f51c1e1d38ad4206ae99cb5afd